### ISSUE TITLE
Unfilled links in discussion page

### ISSUE THREAD

**Issue by ReubenMenezes** [nhcommunity/homepage#41]
### What would you like to share?

The discussion page (https://community.nhcarrigan.com/#discussion) has a few unfilled links.
![image](https://user-images.githubusercontent.com/46624582/150718208-0c110e66-e19f-4c0b-857d-7583d0c3881d.png)

### Additional information

Having a `resources.html` file would be nice as well. And the "Help" link can be https://nhcarrigan.com/community

**Comment by nhcarrigan** [comment]
Yes, we should start compiling a resource list and create the html page.

That help link proposed actually 404s - we could probably reword it to point to the support server.

Also we should remove the `freecodecamp` text - we help with other programming problems too.

**Comment by ReubenMenezes** [comment]
I'd love to take this issue up, may I?

**Comment by nhcarrigan** [comment]
Go for it!

**Comment by ReubenMenezes** [comment]
Thanks!

**Comment by ReubenMenezes** [comment]
Should we close this issue (due to #42)

**Comment by nhcarrigan** [comment]
Yep.
